firmware/ec/common/inc/ocmp_wrappers/ocmp_eeprom_cat24c04.h

```diff
@@ -17,6 +17,7 @@ SCHEMA_IMPORT const Driver_fxnTable CAT24C04_gbc_sid_fxnTable;
 SCHEMA_IMPORT const Driver_fxnTable CAT24C04_gbc_inv_fxnTable;
 SCHEMA_IMPORT const Driver_fxnTable CAT24C04_sdr_inv_fxnTable;
 SCHEMA_IMPORT const Driver_fxnTable CAT24C04_fe_inv_fxnTable;
+SCHEMA_IMPORT const Driver_fxnTable AT45DB641E_fxnTable;
```

Comment:
Suggest these declarations be in alphabetical order

Reply:
As per our current coding guidelines the "#include"s are to be in alphabetical order . If the same needs to be extended for "#define"s as well then we need to change this as part of a separate request as a lot of files might need to be changed .